### ISSUE TITLE
occurrence renderer addon load error from xterm.js v5.1.0.

### ISSUE THREAD

**Issue by mikoto2000** [tsl0922/ttyd#1051]
**Describe the bug**
I tried build and run frontend of 3fcd524df5b985fcf155ae57cf9ca68d0d840033.
But occurred WebGL and Canvas renderer addon load error.

**To Reproduce**
```
# build frontend
cd html/
yarn install
yarn run build

# run devServer and ttyd websocket
npm run start &
ttyd bash &
```

**Expected behavior**
I expect the WebGL addon to load successfully.

**Screenshots**
Developer Tools console log

```
[ttyd] WebGL renderer could not be loaded, falling back to canvas renderer TypeError: Cannot read properties of undefined (reading 'colors')
    at t.WebglAddon.activate (xterm-addon-webgl.js:1:84560)
    at t.AddonManager.loadAddon (xterm.js:1:246631)
    at e.Terminal.loadAddon (xterm.js:1:275269)
    at enableWebglRenderer (index.ts:430:26)
    at Xterm.setRendererType (index.ts:444:17)
    at index.ts:335:26
    at Array.forEach (<anonymous>)
    at Xterm.applyPreferences (index.ts:331:28)
    at Xterm.onSocketData (index.ts:306:22)
index.ts:418 [ttyd] canvas renderer could not be loaded, falling back to dom renderer TypeError: Cannot read properties of undefined (reading 'colors')
    at e.CanvasAddon.activate (xterm-addon-canvas.js:1:78885)
    at t.AddonManager.loadAddon (xterm.js:1:246631)
    at e.Terminal.loadAddon (xterm.js:1:275269)
    at enableCanvasRenderer (index.ts:415:31)
    at enableWebglRenderer (index.ts:435:17)
    at Xterm.setRendererType (index.ts:444:17)
    at index.ts:335:26
    at Array.forEach (<anonymous>)
    at Xterm.applyPreferences (index.ts:331:28)
    at Xterm.onSocketData (index.ts:306:22)
```

**Environment:**
 - OS: `ubuntu:jammy` docker image
     - install nodejs and ttyd build depend packages
 - nodejs: v16.19.0
 - Browser: Firefox 108.0.1, Vivaldi 5.6.2867.50, Google Chrome 108.0.5359.125

**Additional context**
[xterm.js 5.1.0 was released on Dec 20, 2022.](https://github.com/xtermjs/xterm.js/releases/tag/5.1.0)

When I checked `yarn.lock`, the version of xterm was `5.1.0`.

```
-"xterm@npm:^5.0.0":
-  version: 5.0.0
-  resolution: "xterm@npm:5.0.0"
-  checksum: c2f1d02a708d3d02bebf052dfdf54ecd1b619386bed8f27ed42483d398a3e0bc3595fc855c600db99fd3a91f6d3e8feabe118f1f8f14bbed92d8b3d77277ab92
-  languageName: node
-  linkType: hard
...(snip)
+xterm@^5.0.0:
+  version "5.1.0"
+  resolved "https://registry.yarnpkg.com/xterm/-/xterm-5.1.0.tgz#3e160d60e6801c864b55adf19171c49d2ff2b4fc"
+  integrity sha512-LovENH4WDzpwynj+OTkLyZgJPeDom9Gra4DMlGAgz6pZhIDCQ+YuO7yfwanY+gVbn/mmZIStNOnVRU/ikQuAEQ==
```

After changing the `xterm` version in `package.json` from `^5.0.0` to `5.0.0`, it can be loaded.

**Comment by mikoto2000** [comment]
After making the following changes to `package.json`, it seems to work.

```diff
root@3dafdb889333:/ttyd/html# git diff package.json
diff --git a/html/package.json b/html/package.json
index 05d89a8..9ccd0aa 100644
--- a/html/package.json
+++ b/html/package.json
@@ -54,11 +54,11 @@
     "trzsz": "^0.4.3",
     "whatwg-fetch": "^3.6.2",
     "xterm": "^5.0.0",
-    "xterm-addon-canvas": "^0.2.0",
-    "xterm-addon-fit": "^0.6.0",
+    "xterm-addon-canvas": "^0.3.0",
     "xterm-addon-image": "^0.2.0",
-    "xterm-addon-web-links": "^0.7.0",
-    "xterm-addon-webgl": "^0.13.0",
+    "xterm-addon-web-links": "^0.8.0",
+    "xterm-addon-webgl": "^0.14.0",
     "zmodem.js": "^0.1.10"
   },
   "resolutions": {
```
(I guess `xterm` should be `^5.1.0` as well...)

**Comment by mikoto2000** [comment]
Fixed in 6d73c5d98983a90df4401dade4bfebcf529f3d03, Thank you!
